### PR TITLE
How to install internal CA for generating domain controller certificate for PIV credential login

### PR DESCRIPTION
**Pull request by clstmbrly**
@lachellel This is the domain controllers write-up for Issue #109 (PIV-Guides).  If you need this submitted to a different branch, please let me know.

### CODE REVIEW

File: _networkconfig/2_domaincontrollers.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ title: Domain Controllers
 collection: networkconfig
 permalink: networkconfig/domaincontrollers/
 ---
+# How do I generate and install Certification Authority (CA) certificates for Domain Controllers?
 
-To use smartcards and PIV credentials for network authentication, all the Domain Controllers need to have Domain Controller Authentication certificates.
+These procedures are intended for network and system administrators, or other IT professionals, who are responsible for the day-to-day network operations of Federal Government agencies. As part of their roles, these professionals will be authorized by their agencies to manage various aspects of their networks, including generating and installing CA certificates for their Domain Controllers. Domain Controller certificates are required in order for the agencies' authorized users to use PIV card credentials for network authentication.
```
Comment (lachellel) [inline _networkconfig/2_domaincontrollers.md:9]: We seem to be making this the _opposite_ of plain language and speaking to the audience?  I'm removing this.  :I wouldn't add words that aren't needed.

Comment (rt-smithee) [reply _networkconfig/2_domaincontrollers.md:9]: the entire concept of using PIV with Windows with domains is extremely challenging IMO and so for example it's not documented anywhere I know of that you need to issue DC's certificates.   That one sentence has great value in opening one's eyes to the level of complexity involved.

**Comment by clstmbrly** [comment]
@lachellel I appreciate the feedback.  Will not include such introductory information in the future.

File: _networkconfig/2_domaincontrollers.md (hunk 1 above)
Comment (lachellel) [reply _networkconfig/2_domaincontrollers.md:9]: Agree and it's been working fine with the agencies.  Just need the one sentence!  They already know who does what and how things operate inside.

**Comment by lachellel** [comment]
@clstmbrly 

After looking at this - yes, let's move to a branch I created here: 

https://github.com/GSA/piv-guides/tree/localca

And make a separate page just for a local CA setup for domain controller certs.  We can just pull that lower section out to another page.

**Comment by lachellel** [comment]
@clstmbrly no big deal!  It's okay to write something that is focused to the audience.  Talk to the person and they don't need any filler!

**Comment by clstmbrly** [comment]
@lachellel Thanks for the additional feedback.  It's really helpful.  Will take care of splitting this into two pages.  Will submit against the new branch you created.  @godadada

**Comment by clstmbrly** [comment]
Modifications needed.

**Pull request closed without merging**